### PR TITLE
Fix missing NumModelSkins serverside

### PR DESCRIPTION
**Pull request by Python1320**
```lua
-- test:
print(WireLib.NumModelSkins'models/props_junk/plasticCrate01a.mdl')
```

### CODE REVIEW

**Comment by Python1320** [comment]
I tried to find steps, but I don't know what triggers it. 
I just know the code that is in there would never work (since the function does not exist on server) whereas the one added should.

**Comment by Divran** [comment]
merged this now because I got a report that someone got an error about this function being missing

File: lua/wire/server/wirelib.lua [hunk 1]
```diff
@@ -1058,10 +1058,21 @@ function WireLib.dummytrace(ent)
 	}
 end
 
+function WireLib.NumModelSkins(model)
+	if NumModelSkins then
```
Comment (TomyLobo) [inline lua/wire/server/wirelib.lua]: Why the if?
Are there cases where it does exist?
If so, who creates it?
Can we depend on them implementing it correctly now and for all eternity?

Comment (AbigailBuccaneer) [inline lua/wire/server/wirelib.lua]: It exists on the client, but not on the server.

**Pull request merged**